test.js

```diff
@@ -133,6 +138,46 @@ describe('v8flags', function () {
     v8flags(function (err, flags) {
       expect(err).to.be.null;
       done();
-    })
+    });
+  });
+});
+
+describe('config-path', function () {
+  const moduleName = 'js-v8flags';
+
+  beforeEach(function() {
+    env.HOME = 'somehome';
+    cleanup();
+  });
+  afterEach(cleanup);
+
+  it('should return default linux path in other environments', function(done) {
+    delete require.cache[require.resolve('./config-path.js')];
```

Comment:
It's been a while back but I think it does yes. I consider it bad but this pattern is used in other existing tests too. The best way to use this would be in sandboxes but that falls outside the scope of this PR.